client/analytics/report/customers/table.js

```diff
@@ -187,6 +188,30 @@ export default class CustomersReportTable extends Component {
 		} );
 	}
 
+	getSummary( totals ) {
+		if ( ! totals ) {
+			return [];
+		}
+		return [
+			{
+				label: __( 'customers', 'wc-admin' ),
+				value: numberFormat( totals.customers_count ),
+			},
+			{
+				label: __( 'average orders', 'wc-admin' ),
+				value: numberFormat( totals.avg_orders_count ),
```

Comment:
One decimal point is enough I think.

<img width="172" alt="screen shot 2019-01-24 at 3 20 15 pm" src="https://user-images.githubusercontent.com/1922453/51650420-ecf91300-1fec-11e9-9392-5b59d191aa34.png">